backend/src/main/java/pl/edu/agh/awiteks_backend/api/users/UserController.java

```diff
@@ -1,14 +1,18 @@
 package pl.edu.agh.awiteks_backend.api.users;
 
 import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 import pl.edu.agh.awiteks_backend.models.User;
+import pl.edu.agh.awiteks_backend.security.jwt.JwtAccessToken;
 import pl.edu.agh.awiteks_backend.services.UserService;
 
 import java.util.List;
 import java.util.Optional;
 
+import static pl.edu.agh.awiteks_backend.configs.SwaggerConfig.JWT_AUTH;
+
 @RestController
 @RequestMapping("/users")
 public class UserController {
```

Comment:
Czemu pozostałem enpointy nie mają tego oznaczenia `security`? Chyba też powinny być zabezpieczone, żeby nikt nie mógł modyfikować listy użytkowników

Reply:
To się wiąże z moim komentarzem z tego samego pliku (nie wiem jak tu linkować to):
```
    // TODO most of these endpoints are useless for frontend and mustn't be open to it
    // TODO either implement admin auth and make them open only for admin or remove them
```
Nie widzę za bardzo zastosowania na froncie endpointów typu `addUser`, `getAllUsers`, czy też `removeUser`, imo są to bardziej endpointy dla admina (czyli dla nas żeby je sprawdzać jakimś postmanem) i tutaj autentykacja JWT raczej wielkiego sensu nie ma.

Reply:
No dobra, niech zostanie na razie tak jak jest, tylko przed deployem tego gdziekolwiek, trzeba będzie pamiętać, żeby to zabezpieczyć